packages/server-wallet/src/protocols/process-ledger-queue.ts

```diff
@@ -1,11 +1,36 @@
+/**
+ * Follows an algorithm to guarantee that an agreed upon ledger update will be signed by
+ * 2 participants in at most 2 round trips. Either can propose a state at any time and either
+ * can counterpropose a state at any time. However, if they follow the algorithm below they
+ * should always be able to compute the expected outcome by the second round trip.
+ *
+ * Algorithm:
+ *
+ * IF there are any pending ledger updates (to fund OR defund a channel)
+ *
+ *   i. Check if I already proposed a new ledger update O₁. Otherwise, iteratively allocate funds
```

Comment:
How should the requests be ordered?

Reply:
They're sorted by channel nonce (proxy for creation time) at the moment